packages/TelegramClient-Tests.package/TCTAPITests.class/instance/tearDown.st

```diff
@@ -0,0 +1,4 @@
+running
+tearDown
+
+	self core client specificClient isNil ifFalse: [self core client free].
```

Comment:
Does that formatting work with the linter? And is there a reason you test for isNil, whereas other test packages do not?

Reply:
we copied that code from existing code in `TCTLoggedInTests`